kleva-j/iReporter
# Chore/#162502506/setup postgresql database

#### What does this PR do?
- Create a database connection for the app to store data.

#### Description of the task to be completed
- Create a database connection.
- Create a user and incident table.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Chore/#162502506/setup-postgresql-database
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162502506](https://www.pivotaltracker.com/story/show/162502506)
